docs/features/virtual-fields.md

```
@@ -0,0 +1,213 @@
+
+:::
+
+```ts
+@Kavo(Book, {
+  filter: { fields: ["titleLower"] },
+  sort: { fields: ["titleLower"] },
+  select: { fields: ["id", "title", "titleLower"] },
+})
+```
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)